templates/model_inner_enum.mustache

```diff
@@ -0,0 +1,101 @@
+// {{{classname}}}{{{nameInCamelCase}}} {{{description}}}{{^description}}the model '{{{classname}}}{{{nameInCamelCase}}}'{{/description}}
```

Comment:
Maybe you can metion how these templates are different from the default templates.

Reply:
Mention that in the template or in the PR?

Reply:
It's ok as it's written in the PR description